src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,29 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads Serverconfig from File pathToResourceDocker/ssh.rc
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
```

Comment:
on the first point: the .split() returns an array and the [1] is the value of e.g. "p2pip=127.0.0.1" and this is what is needed. it actually goes thru all lines and searches for the two terms in the if-clause. "i" is not needed here.
For the second: this is a great idea, I am not yet used to use logger, but this sound perfect!